### PR TITLE
Add local setup instructions

### LINKED ISSUE #1: docs: Add Local setup instructions
## Description
Add the local setup instructions to sun the project for newcomers

### PR DESCRIPTION
**Pull request by mdPial**
Fixes #1 
I have added instructions to setup the project locally. Please check If I need to change or add anything.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,42 @@ The Project is very beginner friendly to contribute to as well as has a user-fri
 
 ## Local Setup Instructions
 
-To be created....
+Follow these steps to setup project locally:
+- **Fork the repository:** Click the fork button of above.
```
Comment (khushishikhu) [inline README.md:16]:
```suggestion
- **Fork the repository:** Click the fork button.
```

File: README.md [hunk 2]
```diff
@@ -12,7 +12,42 @@ The Project is very beginner friendly to contribute to as well as has a user-fri
...
+![fork](https://user-images.githubusercontent.com/43095489/110275583-87fdad00-7ffb-11eb-9828-1b4eb7026b3e.png)
+- **Clone forked repository:** If you are using *Windows OS* use `Git Bash` to run all commands below.
```
Comment (khushishikhu) [inline README.md:18]:
```suggestion
- **Clone forked repository:** If you are using *Windows OS* use `Git Bash` to run all commands given below.
```

File: README.md [hunk 3]
```diff
@@ -12,7 +12,42 @@ The Project is very beginner friendly to contribute to as well as has a user-fri
...
+    yarn --version
+    #to install yarn
+    npm install --global yarn
+    ```
+- **Install dependencies:**
+    ```bash
+    #move to directory
+    cd periodo/periodo
+    #install the dependency
+    yarn install
```
Comment (khushishikhu) [inline README.md:42]:
```suggestion
    yarn install
    or
    npm install
```

File: README.md [hunk 4]
```diff
@@ -12,7 +12,42 @@ The Project is very beginner friendly to contribute to as well as has a user-fri
...
+    ```
+- **Run the project:**
+    ```bash
+    #run in the development mode
+    yarn dev
```
Comment (khushishikhu) [inline README.md:47]:
```suggestion
    yarn dev
    or 
    npm run dev
```

**Review by khushishikhu — CHANGES_REQUESTED** [review]

**Comment by khushishikhu** [comment]
@mdPial Thank you so much for this. I really liked the way you have written the Instructions. We really appreciate your efforts. I have suggested some changes , Just do add all these. And I will happily merge this PR

**Comment by mdPial** [comment]
Thank you for detailed guidelines. I have updated.

**Review by Aaishpra — APPROVED** [review]

**Comment by Aaishpra** [comment]
Great work @mdPial we will merge this after getting more instructions from GirlsScript side so that you can get your points for this PR😉

**Review by khushishikhu — APPROVED** [review]

**Comment by mdPial** [comment]
Okay. Thank you 😄

**Pull request merged**